### ISSUE TITLE
Any help for CentOS?

### ISSUE THREAD

**Issue by shawe** [MatthewClarkMay/geoip-attack-map#17]
I'm interested to try it on CentOS, any help?

**Comment by mikeesparza** [comment]
I've got it running on CentOS 7. What issues are you running into?

**Comment by shawe** [comment]
It runs, but not works, maybe is not reading the correct files. @mikeesparza do you change any access to logs maybe?

**Comment by mikeesparza** [comment]
nope. not the access...Just changed the log location, flag location & MapBox API token to make sure the map shows up.

**Comment by shawe** [comment]
@mikeesparza what log location do you set? Maybe it's the only one that I need ;)

**Comment by mikeesparza** [comment]
I set the location to where I have syslog-ng placing received logs.  /var/log/{name}.log
